src/pages/_app.tsx

```diff
@@ -47,35 +48,47 @@ const saleorConfig: ConfigInput = { apiUrl, channel: channelSlug };
 
 const notificationConfig = { position: positions.BOTTOM_RIGHT, timeout: 2500 };
 
-const App = ({ Component, pageProps }: AppProps) => (
-  <>
-    <Head>
-      <title>Demo PWA Storefront – Saleor Commerce</title>
-      <link rel="preconnect" href={apiUrl} />
-      <link href="https://rsms.me/inter/inter.css" rel="stylesheet" />
-      <link rel="icon" type="image/png" href="/favicon-36.png" />
-      <link rel="manifest" href="/manifest.json" />
-    </Head>
-    <ThemeProvider theme={defaultTheme}>
-      <AlertProvider
-        template={NotificationTemplate as any}
-        {...notificationConfig}
-      >
-        <ServiceWorkerProvider timeout={serviceWorkerTimeout}>
-          <LocaleProvider>
-            <GlobalStyle />
-            <NextQueryParamProvider>
-              <SaleorProvider config={saleorConfig}>
-                <StorefrontApp>
-                  <Component {...pageProps} />
-                </StorefrontApp>
-              </SaleorProvider>
-            </NextQueryParamProvider>
-          </LocaleProvider>
-        </ServiceWorkerProvider>
-      </AlertProvider>
-    </ThemeProvider>
-  </>
-);
+const App = ({ Component, pageProps }: AppProps) => {
+  React.useEffect(() => {
+    // Remove the server-side injected CSS.
+    const jssStyles = document.querySelector("#jss-server-side");
```

Comment:
why?

Reply:
This is the official documentation from `material ui + next.js`